src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.needbraces;
+
+// xdoc section -- start
+class Example1 {
+  String obj = new String();
+  String value = new String();
+  int counter = 1;
+  int count = 0;
+  int num = 12;
+  public boolean Example1() {
+    if (obj.equals(num)) return true;
+    // violation above, ''if' construct must use '{}'s.'
```

Comment:
in Example3 there is no violation on line 19, so line 20 is not same by content.

please do:
```
    if (obj.equals(num)) return true; // violation above, ''if' construct must use '{}'s.'
```
or 
in Example3:
```
if (obj.equals(num)) return true;
// ok, because IF is not a target of validation
```
or 
in Example3:
```
if (obj.equals(num)) return true;

```

RULE:
if code line is present in one Example on Line X, it should be located on same line in all other Examples. Only comments are changing it content from Example to Example, if line contains only violation comment but violation is not raised, it should be empty line or "// ok, because ...."